docs/source/python/recipe/postgresql_pandas.py

```diff
@@ -0,0 +1,60 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# RECIPE STARTS HERE
+
+#: ADBC is integrated into Pandas_, a popular dataframe library.  Pandas can
+#: use ADBC to read the results of SQL queries against PostgreSQL and other
```

Comment:
```suggestion
#: use ADBC to exchange data with PostgreSQL and other
```

Previous comment assumed you could only read. Open to anything but don't want to make it seem like it only goes one way